src/index.ts

```diff
@@ -27,12 +27,37 @@ const config = {
 	ENVIRONMENT: getInput("environment"),
 	VARS: getMultilineInput("vars"),
 	COMMANDS: getMultilineInput("command"),
+	QUIET_MODE: getInput("quiet"),
```

Comment:
```suggestion
	QUIET_MODE: getBooleanInput("quiet"),
```

You also won't need to cast to Boolean elsewhere with this change.

Reply:
Yeah I tried that and kept getting an error. That was like hours of my life trying to get that function to work lmao 
```
| 
|  ❯ src/index.test.ts  (0 test)
| 
| ⎯⎯⎯⎯⎯⎯ Failed Suites 1 ⎯⎯⎯⎯⎯⎯⎯
| 
|  FAIL  src/index.test.ts [ src/index.test.ts ]
| TypeError: Input does not meet YAML 1.2 "Core Schema" specification: quiet
| Support boolean input list: `true | True | TRUE | false | False | FALSE`
|  ❯ Proxy.getBooleanInput node_modules/@actions/core/src/core.ts:179:9
|  ❯ src/index.ts:31:14
|      29|  VARS: getMultilineInput("vars"),
|      30|  COMMANDS: getMultilineInput("command"),
|      31|  QUIET_MODE: getBooleanInput("quiet", {
|        |              ^
|      32|   required: false,
|      33|   trimWhitespace: true,
|  ❯ src/index.test.ts:2:31
| 
| ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯
| 
|  Test Files  1 failed (1)
|       Tests  no tests
|    Start at  19:37:16
|    Duration  501ms (transform 90ms, setup 0ms, collect 0ms, tests 0ms, environment 0ms, prepare 95ms)
```

Reply:
had you set a `default: false` in `action.yml`? that error is occuring because if the value isn't set, the getter doesn't know what to return. See https://github.com/actions/toolkit/issues/844

Reply:
It was set to `default: 'false'` in the action when I did it, and the default won't take a straight boolean value either. Like I said hours of my life 😆 

<img width="851" alt="Screenshot 2023-08-11 at 4 14 13 PM" src="https://github.com/cloudflare/wrangler-action/assets/27247160/c9d6a9a0-6f1a-4789-8ec0-1a8d3ef753f8">

Reply:
oh, I see. that failure is happening in the vitest tests, which doesn't load input defaults into the environment. you could probably get around it by adding a vitest setup file with `process.env.INPUT_QUIET ??= false`. annoying, but I'd rather us keep our implementation clean using available APIs.